### PR TITLE
feat: add autofocus behavior to web component button

### PR DESCRIPTION
**Pull request by chrisdholt**
## Previous Behavior
Autofocus was enumerated as an attribute but not implemented.

## New Behavior
Autofocus works as expected.

## Related Issue(s)
#32394 

- Fixes #

### CODE REVIEW

File: packages/web-components/src/button/button.ts [hunk 1]
```diff
@@ -258,9 +258,13 @@ export class BaseButton extends FASTElement {
     return true;
   }
 
-  connectedCallback(): void {
+  public connectedCallback(): void {
     super.connectedCallback();
     this.elementInternals.ariaDisabled = `${!!this.disabledFocusable}`;
+
+    if (this.autofocus) {
```
Comment (radium-v) [inline packages/web-components/src/button/button.ts:265]: Doesn't having this in the connectedCallback cause it to always steal focus any time it's added to the DOM?

**Review by radium-v — COMMENTED** [review]
`autofocus` is a global attribute. Shouldn't the behavior be handled by the browser? https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Comment by chrisdholt** [comment]
> `autofocus` is a global attribute. Shouldn't the behavior be handled by the browser? https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus

It should - and it looks like it does, we may need more info from the bug that came in. Do you want to follow up on it? Linked above.

**Pull request closed without merging**